include/hip/hcc_detail/hip_runtime.h

```diff
@@ -161,8 +161,7 @@ extern int HIP_TRACE_API;
 // misc
 #define __HIP_ARCH_HAS_SURFACE_FUNCS__ (0)
 #define __HIP_ARCH_HAS_3DGRID__ (1)
-#define __HIP_ARCH_HAS_DYNAMIC_PARALLEL__ (0)
```

Comment:
I don't think removing this is right either. Applications that were checking this flag is the correct way would break. Can you point me to any sample/application that is only checking for ```__HIP_ARCH_HAS_DYNAMIC_PARALLEL__``` being defined instead of checking it's value is 0?